### PR TITLE
update(JS): web/javascript/reference/global_objects/object/entries

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [Object.entries()@MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Object/entries), [сирці Object.entries()@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/object/entries/index.md)

Нові зміни:
- [mdn/content@fcd80ee](https://github.com/mdn/content/commit/fcd80ee4c8477b6f73553bfada841781cf74cf46)

### CODE REVIEW

File: files/uk/web/javascript/reference/global_objects/map/index.md [hunk 1]
```diff
@@ -244,10 +237,16 @@ console.log(contacts.size); // 1
...
+Ці властивості означені на `Map.prototype` і є спільними для всіх примірників `Map`.
+
+- {{jsxref("Object/constructor", "Map.prototype.constructor")}}
+  - : Функція-конструктор, що створила об'єкт-примірник. Для примірників `Map` початковим значенням є конструктор {{jsxref("Map/Map", "Map")}}.
 - `Map.prototype[@@toStringTag]`
   - : Початкове значення властивості [`@@toStringTag`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) – рядок `"Map"`. Ця властивість використовується в {{jsxref("Object.prototype.toString()")}}.
 - {{jsxref("Map.prototype.size")}}
   - : Повертає кількість пар ключ-значення об'єкта `Map`.
+- `Map.prototype[@@toStringTag]`
+  - : Початковим значенням властивості [`@@toStringTag`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) ж рядок `"Map"`. Вона використовується в {{jsxref("Object.prototype.toString()")}}.
```
Comment (AdriandeCita) [inline files/uk/web/javascript/reference/global_objects/map/index.md:249]:
```suggestion
  - : Початковим значенням властивості [`@@toStringTag`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) є рядок `"Map"`. Вона використовується в {{jsxref("Object.prototype.toString()")}}.
```

**Comment by AdriandeCita** [comment]
Оригінальний вміст: [Map@MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Map), [сирці Map@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/map/index.md)

Нові зміни:
- [mdn/content@d19dc31](https://github.com/mdn/content/commit/d19dc31570f62196a5837be38bd0b11c45e67b05)
- [mdn/content@fcd80ee](https://github.com/mdn/content/commit/fcd80ee4c8477b6f73553bfada841781cf74cf46)

**Review by AdriandeCita — CHANGES_REQUESTED** [review]
Дрібничка

**Comment by undead404** [comment]
Прийняв пропозицію

**Review by AdriandeCita — APPROVED** [review]

**Pull request merged**